backend/src/nodes/impl/pytorch/convert_to_onnx_impl.py

```diff
@@ -0,0 +1,42 @@
+from io import BytesIO
+
+import torch
+from spandrel import ModelDescriptor
+
+
+def convert_to_onnx_impl(
+    model: ModelDescriptor,
+    device: torch.device,
+    use_half: bool = False,
+    input_name: str = "input",
+    output_name: str = "output",
+) -> bytes:
+    # https://github.com/onnx/onnx/issues/654
+    dynamic_axes = {
+        input_name: {0: "batch_size", 2: "height", 3: "width"},
+        output_name: {0: "batch_size", 2: "height", 3: "width"},
+    }
+    dummy_input = torch.rand(1, model.input_channels, 64, 64)
+    dummy_input = dummy_input.to(device)
+
+    if use_half:
+        model.model.half()
```

Comment:
We should probably throw an error here if the model doesn't support FP16. I know that you check this before calling this function in the nodes somewhere, but that's global reasoning. So let's ensure correctness locally as well.